Dockerfile.dapper

```diff
@@ -0,0 +1,16 @@
+FROM quay.io/skopeo/stable:latest
```

Comment:
Could a mutating tag here potentially cause headaches down the line if a change is made that then changes behavior?

Reply:
Also pinned.